armi/utils/tests/test_triangle.py

```diff
@@ -88,7 +87,9 @@ def test_checkIfPointIsInTriangle(self):
         y3 = 0.376
         xP = 0.0
         yP = 0.17
-        # FIXME: generalTriangleInOrOut = triangle.checkIfPointIsInTriangle(xT1, yT1, xT2, yT2, xT3, yT3, xP, yP)
+        generalTriangleInOrOut = triangle.checkIfPointIsInTriangle(
+            xT1, yT1, xT2, yT2, xT3, yT3, xP, yP
+        )
```

Comment:
This cracks me up

Reply:
hashtag fixed

Reply:
lol

Yeah, my guess is the concerning bug was fixed years ago, probably right after this test line was commented out.